djangoevents/schema.py

```diff
@@ -0,0 +1,67 @@
+"""
+Aggregate event avro schema validation
+"""
+
+import avro.schema
+import os
+import stringcase
+
+from avro.io import Validate as avro_validate
+from collections import defaultdict
+from django.conf import settings
+from .settings import CONFIG
+from .domain import list_concrete_aggregates, list_aggregate_events
+
+
+schemas = defaultdict(dict)
+
+
+def load_all_event_schemas():
+    """
+    Initializes aggregate event schemas lookup cache.
+    """
+    for aggregate in list_concrete_aggregates():
+        for event in list_aggregate_events(aggregate_cls=aggregate):
```

Comment:
I think it would be good to be able to switch it of for some events, having it turned on by default. Also somebody might want to override `BaseAggregate` with some subclass - that would be achievable by parametrising this and `list_concrete_aggregates` functions, with default set.

Thinking here about some other possible users, that could possibly want to document only crucial part etc.